BUILD.gn

```diff
@@ -556,6 +556,7 @@ source_set("electron_lib") {
     defines += [
       # Disable warnings for g_settings_list_schemas.
       "GLIB_DISABLE_DEPRECATION_WARNINGS",
+      "USE_X11=1",
```

Comment:
uh, if this define is removed upstream, we should probably not be using it!

Reply:
More information for why i left it in https://github.com/electron/electron/issues/31382#issuecomment-979274107